cmd/compose/build.go

```diff
@@ -67,10 +67,15 @@ func (opts buildOptions) toAPIBuildOptions(services []string) (api.BuildOptions,
 		builderName = os.Getenv("BUILDX_BUILDER")
 	}
 
+	// The build backends do not have a "json" progress; use "plain" instead
```

Comment:
you can use `rawjson` so buildkit will render raw json solver events and you get a clean json stream for all events

Reply:
I created https://github.com/moby/buildkit/issues/4769 since I think it would be better if the `rawjson` output would write one JSON object per line, instead of using multiple lines to show a more human-readable JSON record (that's not really that human readable anyway).